src/integration-test/kotlin/integration/helper/IntegrationHelperUtil.kt

```diff
@@ -103,6 +109,94 @@ class IntegrationHelperUtil {
+        return object : IrohaConfig {
+            override val hostname: String
+                get() = withdrawalConfig.iroha.hostname
+            override val port: Int
+                get() = withdrawalConfig.iroha.port
+            override val creator: String
+                get() = registrationAccount
+            override val pubkeyPath: String
+                get() = withdrawalConfig.iroha.pubkeyPath
+            override val privkeyPath: String
+                get() = withdrawalConfig.iroha.privkeyPath
+        }
+    }
+
+    /** Configuration for notary instance */
+    val notaryConfig = loadConfigs("notary", NotaryConfig::class.java, "/notary.properties")
+
+    /** Test configuration of Notary with runtime dependencies */
+    private fun createNotaryConfig(): NotaryConfig {
+        return mock {
+            on { registrationServiceIrohaAccount } doReturn registrationAccount
+            on { tokenStorageAccount } doReturn tokenStorageAccount
+            on { whitelistSetter } doReturn testConfig.whitelistSetter
+            on { refund } doReturn notaryConfig.refund
+            on { iroha } doReturn notaryConfig.iroha
+            on { ethereum } doReturn notaryConfig.ethereum
+            on { bitcoin } doReturn notaryConfig.bitcoin
+        }
+    }
+
+    /**
+     * Run Notary service instance with integration test environment
+     */
+    fun runNotary() {
+        notary.executeNotary(createNotaryConfig())
+    }
+
+    /** Configuration for withdrawal service instance */
+    private val withdrawalConfig =
+        loadConfigs("withdrawal", WithdrawalServiceConfig::class.java, "/withdrawal.properties")
+
+    /** Test configuration of Withdrawal service with runtime dependencies */
+    private fun createWithdrawalConfig(): WithdrawalServiceConfig {
+        val outerTokenStorageAccount = tokenStorageAccount
+        return object : WithdrawalServiceConfig {
+            override val notaryIrohaAccount: String
+                get() = withdrawalConfig.notaryIrohaAccount
+            override val tokenStorageAccount: String
+                get() = outerTokenStorageAccount
+            override val registrationIrohaAccount: String
+                get() = registrationAccount
+            override val iroha: IrohaConfig
+                get() = createIrohaConfig()
+            override val ethereum: EthereumConfig
+                get() = withdrawalConfig.ethereum
+        }
+    }
+
+    /**
+     * Run Withdrawal service instance with integration test environment
+     */
+    fun runWithdrawal() {
+        withdrawalservice.executeWithdrawal(createWithdrawalConfig(), passwordConfig)
+    }
+
+    /** Configuration for registration instance */
+    val registrationConfig =
+        loadConfigs("registration", RegistrationConfig::class.java, "/registration.properties")
+
+    /** Test configuration of Registration with runtime dependencies */
+    private fun createRegistrationConfig(): RegistrationConfig {
```

Comment:
mock looks much better. can we refactor other configuration creating methods to use mocks?